apps/include/opt.h

```diff
@@ -132,7 +132,7 @@
         { "xchain_build", OPT_X_CHAIN_BUILD, '-', \
             "build certificate chain for the extended certificates"}, \
         { "xcertform", OPT_X_CERTFORM, 'F', \
-            "format of Extended certificate (PEM or DER) PEM default " }, \
+            "format of Extended certificate (PEM/DER/P12) PEM default " }, \
```

Comment:
nit: you pushed a ; before `PEM default` in other edits

Reply:
Added the ';' (and updated the text following it).